sechub-cli/src/mercedes-benz.com/sechub/cli/config.go

```diff
@@ -379,12 +393,25 @@ func validateWaitTimeOrWarning(waitTime int) int {
 	// Verify wait time and ensure MinimalWaitTimeSeconds
 	if waitTime < MinimalWaitTimeSeconds {
 		sechubUtil.LogWarning(
-			fmt.Sprintf("Desired wait intervall (%d s) is too small. Setting to %d seconds.", waitTime, MinimalWaitTimeSeconds))
+			fmt.Sprintf("Desired wait interval (%d s) is too small. Setting to %d seconds.", waitTime, MinimalWaitTimeSeconds))
```

Comment:
```suggestion
			fmt.Sprintf("Desired wait interval (%d s) is too small. Setting it to %d seconds.", waitTime, MinimalWaitTimeSeconds))
```

Reply:
changed (also used "short" instead of "small" as proposed in next section)